UnitedIncome/components
# Atoms/Icons -> typescript

### Description
> Please provide a brief description of your changes below.

Converted all of the icons to typescript, updated all tests to use testing-library-react, and updated all of the pathnames. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

make sure conversion is 1 to 1 for all components.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

N/A

### Comments
> Any additional comments that will aid with the review process.
